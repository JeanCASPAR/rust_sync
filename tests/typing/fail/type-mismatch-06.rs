use rustre::sync;

sync! {
    #![pass(1)]
    
    node f() = (x)
    where
        x : int = 3;

    node g() = ()
    where
        () = f();
}

fn main() {}
