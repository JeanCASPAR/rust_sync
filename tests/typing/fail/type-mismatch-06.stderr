error: type mismatch: left has type `unit`, right has type `int`
  --> tests/typing/fail/type-mismatch-06.rs:12:14
   |
12 |         () = f();
   |              ^^^
